.rubocop.yml

```diff
@@ -83,7 +83,7 @@ Metrics/AbcSize:
   Max: 20
 
 Metrics/MethodLength:
-  Max: 15
+  Max: 17
```

Comment:
Since there are a lot of commands it is difficult to accomodate. I will try to use hash to store the flag commands and use loops to print in future.